lib/template/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+postgres:
+  image: postgres
+web:
+  build: .
+  command: bundle exec foreman start
```

Comment:
Ah, I see. Do you really want all of the processes run within the web container as opposed to making separate containers for each entry in Procfile?